examples/plot_multi_axes.py

```diff
@@ -0,0 +1,121 @@
+"""
+Test Plots with Multiple Axes in Matplotcheck
+=============================================
+
+Below you will find some examples of how to use MatPlotCheck
+to test Matplotlib objects with more than one axes in Python.
+
+"""
+
+################################################################################
+# Import Packages
+# ---------------
+# To begin, import the libraries that you need.
+
+import matplotcheck.base as pt
+import matplotlib.pyplot as plt
+import pandas as pd
+
+###############################################################################
+# Setup
+# -----
+# First you must create some example data to plot over multiple axes. This
+# data could represent many different types of plots, such as plotting multiple
+# satellite bands, plotting NDVI of multiple study sites over time, and much
+# more.
+# Once you have created your plot, you will create a Matplotcheck
+# ``PlotTester`` object by providing each Matplotlib axes object to a unique
```

Comment:
Change to: ``PlotTester`` object for each Matplotlib axis object using the `PlotTester()` function.